feature/platform/transceiver/zr_inventory_test/README.md

```diff
@@ -0,0 +1,51 @@
+# TRANSCEIVER-7: Telemetry: 400ZR module inventory information. 
+
+## Summary
+
+Validate 400ZR modules report inventory information part number and serial
+number.
+
+## Procedure
+
+*   Plug in the ZR module in the host port and make sure the transceiver 
+    state is enabled and host is able to detect the module.
+*   With the module recognized verify it reports correct inventory
+    information by subscribing ON_CHANGE to the following telemetry paths.
+
+    *   /platform/components/component/state/serial-no
+    *   /platform/components/component/state/part-no
+
+*   Validate the streamed inventory information data is of type String.
+
+*   Verify that the modules inventory information is reported correctly after
+    an optic software reset.
+
+    *   With ZR module plugged in the host and properly recognized 
+    *   Verify the ZR optics inventory is correctly reported via the 
+        streaming telemetry paths above.
+    *   Reset the optic through software.
+    *   Verify the ZR optics still reports correct inventory information.
+    *   Telemetry subscription should be ON_CHANGE and streamed data should
+        be of type String.
+
+*   Verify that the modules inventory information is reported correctly when
+    interface and transceiver states are disabled.
+
+    *   With ZR module plugged in the host and properly recognized
+    *   Use /components/component/transceiver/config/enabled and 
```

Comment:
This should power off the transceiver according to OC .  Can you specify what the correct response should be for telemetry?  Should  the transceiver should appear to be removed when disabled (powered off)?   

Do you want to test both /components/component/transceiver/config/enabled	 and /interfaces/interface/config/enabled. ?

Reply:
I have split this into two separate tests.

In case of /interfaces/interface/config/enabled set as false inventory info should be reported

In case of /components/component/transceiver/config/enabled set as false module will be powered off and no inventory info will be reported